client/core/trade.go

```diff
@@ -1001,6 +1022,10 @@ func (t *trackedTrade) processAudit(msgID uint64, audit *msgjson.Audit) error {
 		return errs.add("swap contract value %d was lower than expected %d", auditInfo.Coin().Value(), auditQty)
 	}
 
+	// TODO: Consider having the server supply a fee rate: If this is taker,
+	// apply fee rate to their swap; if this is maker, apply fee rate to their
+	// redeem.
+
```

Comment:
Is this comment outdated? The server is supplying both fee rates, I think.

Reply:
Mostly outdated, but in `processAudit`this still refers to possible checks that client can do on the counter party's swap txn.  I've revised the comment in `processAudit` to read as follows:

```go
// TODO: Consider having the server supply the contract txn's fee rate to
// improve the taker's audit with a check of the maker's contract fee rate.
// The server should be checking the fee rate, but the client should not
// trust it. The maker could also check the taker's contract txn fee rate,
// but their contract is already broadcasted, so the check is of less value
// as they can only wait for it to be mined to redeem it, in which case the
// fee rate no longer matters, or wait for the lock time to expire to refund.
```